tools/bump-version.sh

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env bash
+
+# Copyright 2023 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+set -eu -o pipefail
+shopt -s lastpipe
+
+FC_TOOLS_DIR=$(dirname $(realpath $0))
+source "$FC_TOOLS_DIR/functions"
+FC_ROOT_DIR=$FC_TOOLS_DIR/..
+
+
+if [ $# -ne 1 ]; then
+    cat <<EOF
+$0 <version>
+
+    Example: $0 1.4.0-dev
+
+    Bump Firecracker release version:
+    1. Updates Cargo.toml / Cargo.lock
+    2. Runs `cargo update`
```

Comment:
hm, for some reason the `cargo update` piece is evaluated when running the usage:
```
$ ./tools/bump-version.sh
warning: patch for `kvm-bindings` uses the features mechanism. default-features and features will not take effect because the patch dependency does not support this mechanism
    Updating git repository `https://github.com/firecracker-microvm/kvm-bindings`
    Updating crates.io index
    Updating git repository `https://github.com/firecracker-microvm/micro-http`
    Updating proc-macro2 v1.0.56 -> v1.0.57
./tools/bump-version.sh <version>

    Example: ./tools/bump-version.sh 1.4.0-dev

    Bump Firecracker release version:
    1. Updates Cargo.toml / Cargo.lock
    2. Runs
```

Reply:
oh the backticks get expanded X_X. Will fix.